src/ekobox/info_file.py

```diff
@@ -68,19 +68,13 @@ def build(
         hqm_scheme=theory_card.heavy.masses_scheme,
         thresholds_ratios=np.power(list(iter(theory_card.heavy.matching_ratios)), 2.0),
     )
-    alphas_values = np.array(
-        [
-            4.0
-            * np.pi
-            * sc.a_s(
-                muf2,
-            )
-            for muf2 in operators_card.mu2grid
-        ],
-        dtype=float,
-    )
-    template_info["AlphaS_Vals"] = alphas_values.tolist()
-    template_info["AlphaS_Qs"] = np.array(
-        [mu for mu, _ in operators_card.mugrid]
-    ).tolist()
+
+    alphas_values = []
+    alphas_qs = []
+    for mu, nf in operators_card.mugrid:
```

Comment:
there might be lurking another bug around:
- see the counter part for the EKOs [here](https://github.com/NNPDF/eko/blob/38cc28c5c6c646118b674a51023b7a816d1d080e/src/ekobox/evol_pdf.py#L98)
- since we are having tuples there is no natural sorting any longer implied - moreover, here you are relying on the opcard which may have an arbitrary order (as it comes from the user)

thus I suggest a stronger check:
- we first sort the `mugrid` by `nf`and then `mu` (as done for the EKOs)
- we check that we can actually dump to LHAPDF, i.e. there are no conflicting (=overlapping) mu ranges
- the edge case of one overlapping mu point needs to be checked with LHAPDF, i.e. it allows such a thing and does not crash on reading - then we need to decide something, e.g. keep them as suggested by @scarlehoff

Reply:
> since we are having tuples there is no natural sorting any longer implied - moreover, here you are relying on the opcard which may have an arbitrary order (as it comes from the user)

This version is also more likely to have the correct order than the previous one (which was taking the Qs and the values of alpha_s independently)

> we check that we can actually dump to LHAPDF, i.e. there are no conflicting (=overlapping) mu ranges
> the edge case of one overlapping mu point needs to be checked with LHAPDF

All our PDFs from 3.1 (at least) had that overlapping point so it should be fine.

That said, feel free to change it around. I've used this for `NNPDF40MC_nnlo_as_01180` if you want to have a look a PDF with different values of alpha_s for the same Q:

```
 [4]: from lhapdf import mkPDF
   ...: aa = mkPDF("NNPDF40MC_nnlo_as_01180")
   ...: print(aa.alphasQ(1.51))
   ...: print(aa.alphasQ(1.51+1e-13))
   ...: print(aa.alphasQ(1.51-1e-13))

0.34671367444940654
0.3467136744493946
0.3467136744494207
```

Reply:
@scarlehoff can you please have a look to cb6953df if that makes sense?